docs/docs/tutorials/index.md

```diff
@@ -13,6 +13,7 @@ Welcome to the LangGraph Tutorials! These notebooks introduce LangGraph through
 Learn the basics of LangGraph through a comprehensive quick start in which you will build an agent from scratch.
 
 - [Quick Start](introduction.ipynb): In this tutorial, you will build a support chatbot using LangGraph.
+- [LangGraph Platform Quick Start](../cloud/quick_start.md): In this tutorial, you will learn how to deploy a tool-calling chatbot agent to LangGraph Platform.
```

Comment:
https://diataxis.fr/tutorials/#show-the-learner-where-they-ll-be-going

![image](https://github.com/user-attachments/assets/b30d2b0a-97aa-4c5c-8ffd-595f908d889e)


`you will` -> `we will`

Instead of "learn" say "build"

Reply:
ha! this is great, thank you